api/MediaDevices.json

```diff
@@ -29,10 +29,10 @@
             "version_added": "30"
           },
           "safari": {
-            "version_added": "11"
+            "version_added": "13"
```

Comment:
Can you ensure that `navigator.mediaDevices` was also not in Safari 11? Or is this a case of missing interface object?

Reply:
Or better still, can you update api.Navigator.mediaDevices to match, in this PR?

Reply:
@vinyldarkscratch what ended up happening here? There are still changes in MediaDevices.json, but not these ones. Is there an issue with the methodology that lead to "13" originally?

Reply:
The `MediaDevices` interface wasn't directly exposed until Safari 13, so this is another case of needing to test an instance rather than the interface.  `navigator.mediaDevices` was present in Safari 11.